http://www.vdr-portal.de/board/thread.php?postid=660980#post660980
fix some missing epg infos on any Premiere-Direct feeds
diff -Naur -- premiereepg-0.0.8.orig/premiereepg.c premiereepg-0.0.8/premiereepg.c
--- premiereepg-0.0.8.orig/premiereepg.c	2007-12-26 11:29:45.611133534 +0100
+++ premiereepg-0.0.8/premiereepg.c	2007-12-26 11:31:38.310054659 +0100
@@ -626,6 +626,7 @@
 	      else if(tid==0x03 && sid==0xf5) { tid=0x03; sid=0xdc; }
 	      else if(tid==0x04 && sid==0xd2) { tid=0x11; sid=0xe2; }
 	      else if(tid==0x11 && sid==0xd3) { tid=0x11; sid=0xe3; }
+          else if(tid==0x01 && sid==0xd4) { tid=0x04; sid=0xe4; }
               }
             }
           tChannelID channelID(Source(),nid,tid,sid);
